### PR TITLE
Initial Commit with logo,button,googlrbutton,vontatiner,stylehelper

### PR DESCRIPTION
**Pull request by GauriBane16**
Need feedback on this

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -13,20 +13,21 @@
   }
 }
 
+
+
 .App-header {
-  background-color: white; /* #282c34 */
```
Comment (ShashankSuresh) [inline src/App.css:17]: Colors can be importing from CSS like this https://css-tricks.com/getting-javascript-to-talk-to-css-and-sass/

File: src/App.css [hunk 2]
```diff
@@ -13,20 +13,21 @@
...
+  background-color: #282c34;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
-  justify-content: flex-start;
+  align-items: center;
+  justify-content: center;
   font-size: calc(10px + 2vmin);
```
Comment (ShashankSuresh) [inline src/App.css]: Font sizes will be constant - they can be imported from CSS variables as well

File: src/App.css [hunk 3]
```diff
@@ -13,20 +13,21 @@
...
-  font-weight: bold;
   color: white;
 }
 
 .App-link {
-  color: #2592aa;
+  color: #61dafb;
```
Comment (ShashankSuresh) [inline src/App.css]: Colors to come from CSS variable file

File: src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import Login from './Pages/Login';
+
+
```
Comment (ShashankSuresh) [inline src/App.jsx]: Please remove unwanted new lines

File: src/Pages/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import GoogleLogin from "../library/components/GoogleLogin";
+import Logo from "../library/components/Logo";
+import CLabel from "../library/components/Label";
+import Content from "../library/components/Content";
+
+const Login = () => {
+  return (
+    <Content className="MainContainer" >
+        <Content style={{ margin: "3rem 0 2rem 0", textAlign: "center" }}>
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: Lets avoid using inline styling. Either use StyledComponents or add a CSS class to it

File: src/Pages/Login.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      <Logo width={243} height={50}/>
+      <CLabel>
+        <h6 style={{ fontStyle: "italic", margin: 0 }}>
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: Same here - avoid using inline styling

File: src/Pages/Login.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+          <em>We are at your service</em>
+        </h6>
+      </CLabel>
+      </Content>
+      <Content className="Container"
+        style={{
+          height: "60vh",
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: Same here - avoid using inline styling

File: src/Pages/Login.jsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    <Content className="MainContainer" >
+        <Content style={{ margin: "3rem 0 2rem 0", textAlign: "center" }}>
+      <Logo width={243} height={50}/>
+      <CLabel>
+        <h6 style={{ fontStyle: "italic", margin: 0 }}>
+          <em>We are at your service</em>
+        </h6>
+      </CLabel>
+      </Content>
+      <Content className="Container"
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: You are using css class Container and inline style. Lets stick to one.

File: src/Pages/Login.jsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+          textAlign: "center"
+        }}
+      >
+        <CLabel>
+          <h2>Sign In</h2>
+        </CLabel>
+        <CLabel>
+          <h6 style={{ margin: 0 }}>Welcome @DoorStep!</h6>
+        </CLabel>
+        <GoogleLogin>Login For Admin</GoogleLogin>
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx:25]: This can be loaded from a config file in local and use a map to iterate through it

File: src/Pages/Login.jsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+          width: "50vw",
+          margin: 0,
+          textAlign: "center"
+        }}
+      >
+        <CLabel>
+          <h2>Sign In</h2>
+        </CLabel>
+        <CLabel>
+          <h6 style={{ margin: 0 }}>Welcome @DoorStep!</h6>
```
Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: Same here - avoid using inline styling

Comment (ShashankSuresh) [inline src/Pages/Login.jsx]: Also component Headings should be exporting headings from h1 to h6. We dont have to explicitly use h6 here

File: src/index.css [hunk 11]
```diff
@@ -5,6 +5,15 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  /* background-image: url('./library/images/background.jpg'); */
```
Comment (ShashankSuresh) [inline src/index.css]: Remove these 2 lines if not required

File: src/library/components/Buttons.jsx [hunk 12]
```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```
Comment (ShashankSuresh) [inline src/library/components/Buttons.jsx:1]: Good job on this component. :)

File: src/library/components/Content.jsx [hunk 13]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
```
Comment (ShashankSuresh) [inline src/library/components/Content.jsx:1]: Good job on this component. :)

File: src/library/components/GoogleLogin.jsx [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+  box-shadow: none !important;
+  border: 1px solid ${Colors.primary}!important;
+  color: ${Colors.primary}!important;
+  border-radius: 14px !important;
+  justify-content: center; ;
+`;
+
+const GLogin = ({ children }) => {
+  const clientId =
+    "737423385254-lolcijd6n4834nn94heseis2gptsashc.apps.googleusercontent.com";
```
Comment (ShashankSuresh) [inline src/library/components/GoogleLogin.jsx]: Such sensitive keys should be loaded from env files.

File: src/library/components/Logo.jsx [hunk 15]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import logo from "../images/logo.png";
+
+
+const Logo = (props) => {
+  return (
```
Comment (ShashankSuresh) [inline src/library/components/Logo.jsx]: The return keyword is not required

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (GauriBane16) [reply src/App.css:17]: we are not using this class, this is default created with create-react-app.
I will remove this.
thanks for link

File: src/App.css (hunk 2 above)
Comment (GauriBane16) [reply src/App.css]: Not Using. It's default . I will remove this

File: src/App.css (hunk 3 above)
Comment (GauriBane16) [reply src/App.css]: Not using. default with boiler plate via create-react-app

File: src/library/components/Buttons.jsx (hunk 12 above)
Comment (GauriBane16) [reply src/library/components/Buttons.jsx:1]: Thanks

File: src/library/components/GoogleLogin.jsx (hunk 14 above)
Comment (GauriBane16) [reply src/library/components/GoogleLogin.jsx]: Yes, I will

File: src/library/components/Content.jsx (hunk 13 above)
Comment (GauriBane16) [reply src/library/components/Content.jsx:1]: Thanks

File: src/library/components/Logo.jsx (hunk 15 above)
Comment (GauriBane16) [reply src/library/components/Logo.jsx]: Ok.Will follow this

File: src/App.jsx (hunk 4 above)
Comment (GauriBane16) [reply src/App.jsx]: yes

File: src/Pages/Login.jsx (hunk 5 above)
Comment (GauriBane16) [reply src/Pages/Login.jsx]: ok

File: src/index.css (hunk 11 above)
Comment (GauriBane16) [reply src/index.css]: yes

File: src/Pages/Login.jsx (hunk 9 above)
Comment (gauribane) [reply src/Pages/Login.jsx:25]: Not Getting this point

File: src/Pages/Login.jsx (hunk 8 above)
Comment (gauribane) [reply src/Pages/Login.jsx]: Done

File: src/Pages/Login.jsx (hunk 7 above)
Comment (gauribane) [reply src/Pages/Login.jsx]: yes

File: src/Pages/Login.jsx (hunk 6 above)
Comment (gauribane) [reply src/Pages/Login.jsx]: ok

File: src/Pages/Login.jsx (hunk 10 above)
Comment (gauribane) [reply src/Pages/Login.jsx]: yes

Comment (gauribane) [reply src/Pages/Login.jsx]: Need more input on this

**Pull request merged**